modules/efk-logging-fluentd-log-location.adoc

```diff
@@ -3,9 +3,9 @@
 // * logging/efk-logging-fluentd.adoc
 
 [id="efk-logging-fluentd-log-location_{context}"]
-= Configuring Fluentd log location
+= Configuring the collected log location
 
-Fluentd writes logs to a specified file or to the default location, `/var/log/fluentd/fluentd.log`, 
+The log collector, Fluentd or Rsyslog, writes logs to a specified file or to the default location, `/var/log/fluentd/fluentd.log`, 
```

Comment:
For fluentd, it's /var/log/fluentd/fluentd.log , for rsyslog, it is /var/log/rsyslog/rsyslog.log